client/jetpack-onboarding/steps/homepage.jsx

```diff
@@ -21,6 +21,7 @@ import { saveJetpackOnboardingSettings } from 'state/jetpack-onboarding/actions'
 class JetpackOnboardingHomepageStep extends React.PureComponent {
 	handleHomepageSelection = homepageFormat => {
 		const { siteId } = this.props;
+		this.props.recordJpoEvent( 'calypso_jpo_' + homepageFormat + '_clicked' );
```

Comment:
Nit: I usually prefer leaving an empty line between the `const`/`let` definitions and the rest of the code below.